artifact/image/unpack/unpack_test.go

```diff
@@ -26,9 +26,10 @@ import (
 	"testing"
 
 	"archive/tar"
+
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
-	"github.com/google/go-containerregistry/pkg/v1"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
```

Comment:
Not quite sure if I understand why it's better formatting to make things ending with "v1" a named import?

Reply:
I was curious about that myself but unfortunately I couldn't find a concrete laymen answer, just the [issue that lead to it being a thing](https://github.com/golang/go/issues/29041).

From what I could tell, _something_ has an issue with implicitly-named imports of versions like this maybe because of special handling Go has for major versions of packages, and that resulted in something erroring - I think it might be a technical edge case in `goimports` rather than an actual language constraint, and that they decided it was fine to just make the convention to explicitly define the package name in this case?